compiler/ballerina-lang/src/test/java/org/ballerinalang/toml/ManifestProcessorTest.java

```diff
@@ -154,10 +165,23 @@ public void testMultipleDependencies() throws TomlException, IOException {
         Path tmpDir = Files.createTempDirectory("manifest-test-");
         Path baloPath = tmpDir.resolve("string_utils.balo");
         Files.createFile(baloPath);
-        
-        Manifest manifest = ManifestProcessor.parseTomlContentFromString(this.validProjectBlock + "[dependencies] \n " +
-                "string-utils = { path = \"" + baloPath + "\", version = \"1.0.5\" } \n " +
-                "jquery = { version = \"2.2.3\" } \n");
+
+        String baloPathStr = baloPath.toString();
+        String tomlData;
+
+        if (baloPathStr.contains("/")) {
+            tomlData = this.validProjectBlock + "[dependencies] \n " +
```

Comment:
Same here, We can simplify this logic. This line is redundant in both cases.

Reply:
done